### PR TITLE
fix(backend): removed settings hacks

### PR DESCRIPTION
**Pull request by ak239**
We used to create custom user data dir for each run, so we can run
more then one ndb window at the same time. We can achieve the same
result by reusing the same app with multiple windows.

- removed robust user data dir handling,
- cleanup settings related code,
- added couple todos for followup.

drive-by: fixed terminal.

### CODE REVIEW

File: front_end/ndb/InspectorFrontendHostOverrides.js [hunk 1]
```diff
@@ -5,32 +5,21 @@
...
 (function(){
-  let hostBackend;
-
   InspectorFrontendHost.getPreferences = async function(callback) {
     [Ndb.backend] = await carlo.loadParams();
-    const info = await getProcessInfo();
-    hostBackend = await Ndb.backend.createService('inspector_frontend_host.js', info.configDir);
-    callback(await hostBackend.getPreferences());
+    const prefs = {
+      '__bundled__panel-selectedTab': '"sources"',
```
Comment (pavelfeldman) [inline front_end/ndb/InspectorFrontendHostOverrides.js]: Let's make these query params upstream.

File: package.json [hunk 2]
```diff
@@ -41,6 +39,7 @@
     "chrome-devtools-frontend": "1.0.612830",
     "eslint": "^5.9.0",
     "mocha": "^5.2.0",
+    "rimraf": "^2.6.2",
```
Comment (pavelfeldman) [inline package.json]: Do you need this?

Comment (ak239) [reply package.json]: Yes, it is dev dependency and required for build script.

Comment (pavelfeldman) [reply package.json]: still lgtm

**Pull request closed without merging**